assembl/static2/js/app/components/home/header.jsx

```diff
@@ -24,40 +27,51 @@ class Header extends React.Component {
   }
 
   render() {
-    const { debateData } = this.props.debate;
-    const { locale } = this.props.i18n;
-    const { timeline } = this.props;
+    const {
+      timeline,
+      data: { discussion: { title, subtitle, headerImage, logoImage, buttonLabel, startDate, endDate } }
+    } = this.props;
     return (
       <section className="home-section header-section">
         <Grid fluid className="max-container">
           <div className="header-content">
-            {debateData.headerLogoUrl ? <img className="header-logo" src={debateData.headerLogoUrl} alt="logo" /> : null}
+            {logoImage && logoImage.externalUrl ? <img className="header-logo" src={logoImage.externalUrl} alt="logo" /> : null}
             <div className="max-text-width">
-              {debateData.topic && <h1 className="light-title-1">{debateData.topic.titleEntries[locale]}</h1>}
+              {title && <h1 className="light-title-1">{title}</h1>}
```

Comment:
```{title ? <h1 className="light-title-1">{title}</h1> : null}``` is now the preferred syntax

Reply:
No, we probably should but they are a few on which people disagree. I think this one is not a stylistic choice but has a reason in terms of code logic